pdal/KDIndex.hpp

```diff
@@ -373,6 +373,251 @@ class PDAL_DLL KD3Index : public KDIndex<3>
         return radius(x, y, z, r);
     }
 
+};
+
+class PDAL_DLL KDNormalIndex : public KDIndex<3>
```

Comment:
I think I'll take this back out. It is not used in this PR and is a result of me learning how to work with nanoflann to create additional index adapters.